### PR TITLE
explain fixes

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
**What problem is the PR solving / What's new?**
Related to #701 

**Checklist**
- [x] I made a PR to develop (not master)
- [ ] If applicable: I updated the docs
- [ ] I updated the CHANGELOG
- [x] These changes are covered by the tests

### CODE REVIEW

File: seq2science/rules/configuration_explain.smk [hunk 1]
```diff
@@ -108,7 +108,7 @@ else:
             assert isinstance(ret,dict), f"Invalid config key '{opt}' in {opts}."
             ret = ret.get(opt)
             if ret in [None, ""]:
-                return ""
+                return " with default options"
```
Comment (Maarten-vd-Sande) [inline seq2science/rules/configuration_explain.smk:111]: @siebrenf would this work, or are there edge cases I don't think of? (and the tests miss those)

**Review by siebrenf — APPROVED** [review]
